internal/pkg/service/buffer/worker/distribution/node.go

```diff
@@ -152,6 +152,9 @@ func (n *Node) onWatchErr(err error) {
 }
 
 func (n *Node) createSession() (err error) {
+	startTime := time.Now()
+	n.logger.Infof(`creating etcd session`)
+
```

Comment:
Pridal som logovanie toho ako dlho operacia trvala.

Ked bude niekedy cas a bude sa migrovat na DD, tak budeme namiesto textu logovat JSON,
a tam spravime nejaky field `duration`, ale zatial to mame takto.

To lepsie logovanie mame pripravene, zapisalo by sa to takto:
```
n.logger.With("duration", time.Since(startTime)).Infof(....)`
```

Co nie je vyriesene, je ako to naformatovat ako text,
v aktualnom style logovania.